com.ibm.streamsx.inet/com.ibm.streamsx.inet.ftp/FTPCommand/FTPCommand.xml

```diff
@@ -53,14 +53,61 @@ The command string and the command arguments are received from port 0.
             <prototype><![CDATA[rstring Url()]]></prototype>
           </function>
           <function>
-            <description>Get the number of executed ftp commands(in case of rename only one is counted)</description>
+            <description>Deprecated: Get the number of executed ftp commands(in case of rename only one is counted)</description>
             <prototype><![CDATA[uint32 NoCommands()]]></prototype>
           </function>
           <function>
-            <description>Get the number of failed ftp commands</description>
+            <description>Get the number of executed ftp commands(in case of rename only one is counted)</description>
+            <prototype><![CDATA[uint32 CommandCount()]]></prototype>
+          </function>
+          <function>
+            <description>Deprecated: Get the number of failed ftp commands</description>
             <prototype><![CDATA[uint32 NoFailures()]]></prototype>
           </function>
           <function>
+            <description>Get the number of failed ftp commands</description>
+            <prototype><![CDATA[uint32 CommandFailureCount()]]></prototype>
+          </function>
+          <function>
+            <description>Return the argument unchanged.</description>
+            <prototype><![CDATA[<any T> T AsIs(T)]]></prototype>
+          </function>
+        </customOutputFunction>
+        <customOutputFunction>
+          <name>GetError</name>
+          <function>
+            <description>Deprecated: Get the error number</description>
+            <prototype><![CDATA[int32 Error()]]></prototype>
+          </function>
+          <function>
+            <description>Get the error number</description>
+            <prototype><![CDATA[int32 ErrorCode()]]></prototype>
+          </function>
+          <function>
+            <description>Get the error description</description>
+            <prototype><![CDATA[rstring ErrorText()]]></prototype>
+          </function>
+          <function>
+            <description>This function returns a string with the url including schema.</description>
+            <prototype><![CDATA[rstring Url()]]></prototype>
+          </function>
+          <function>
+            <description>Deprecated: Get the number of executed ftp commands(in case of rename only one is counted)</description>
+            <prototype><![CDATA[uint32 NoTransfers()]]></prototype>
+          </function>
+          <function>
+            <description>Get the number of executed ftp commands(in case of rename only one is counted)</description>
+            <prototype><![CDATA[uint32 CommandCount()]]></prototype>
```

Comment:
This is not wrong. In line 61 the function declaration is group 'GetData'. In line 100 the declaration is function group 'GetError'.